debian/changelog

```diff
@@ -1,3 +1,17 @@
+ubuntu-advantage-tools (27.6~22.04.1) jammy; urgency=medium
+
+  * d/tools.postinst:
+    - make log files world readable
+  * New upstream release 27.6 (LP: #1958556)
+    - cli: only go for resources on explicit help calls
```

Comment:
"go for resources" makes sense to me but likely won't for someone outside the project
```suggestion
    - cli: only request available resources from contract server when necessary
```